packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -934,6 +943,21 @@ msgstr "Při nahrávání souboru došlo k chybě: %message%"
 msgid "Error occurred, try again please."
 msgstr "Nastala chyba, zkuste to, prosím, znovu."
 
+msgid "Every 5 minutes"
+msgstr "Každých 5 minut"
+
+msgid "Every 5 minutes in %hour% hour"
+msgstr "Každých 5 minut v %hour% hodinu"
```

Comment:
Is not there an option to use different translations based on the parameter value?

e.g.
for `%hour% = 2` -> `Každých 5 minut ve 2 hodiny`
for `%hour% = 5` -> `Každých 5 minut v 5 hodin`